packages/core/src/api/exporters/markdown/__snapshots__/complex/misc/markdown.md

```diff
@@ -1,4 +1,4 @@
-## **Heading ***~~2~~*
+## **Heading&#x20;***~~2~~*
```

Comment:
Is this correct? I know that it's a unicode space character but it doesn't seem quite right to have in the Markdown

Reply:
I think this might be done to prevent markdown parsers from trimming text before the space